geopandas/io/tests/test_sql.py

```diff
@@ -75,6 +76,95 @@ def connection_spatialite():
     con.close()
 
 
+def connect_engine(dbname, user=None, password=None, host=None, port=None):
+    """
+    Initiaties a connection (engine) to a postGIS database that must already exist.
+    See create_postgis for more information.
+    """
+    sqlalchemy = pytest.importorskip("sqlalchemy")
+    from sqlalchemy.engine.url import URL
+
+    user = user or os.environ.get("PGUSER")
+    password = password or os.environ.get("PGPASSWORD")
+    host = host or os.environ.get("PGHOST")
+    port = port or os.environ.get("PGPORT")
+    try:
+        con = sqlalchemy.create_engine(
+            URL(
+                drivername="postgresql+psycopg2",
+                username=user,
+                database=dbname,
+                password=password,
+                host=host,
+                port=port,
+            )
+        )
+        con.begin()
+    except Exception:
+        return None
+
+    return con
+
+
+def drop_table_if_exists(engine, table):
+    sqlalchemy = pytest.importorskip("sqlalchemy")
+    from sqlalchemy.ext.declarative import declarative_base
+
+    if engine.has_table(table):
+        base = declarative_base()
+        metadata = sqlalchemy.MetaData(engine)
+        metadata.reflect()
+        table = metadata.tables.get(table)
+        if table is not None:
+            base.metadata.drop_all(engine, [table], checkfirst=True)
```

Comment:
```suggestion
            table.drop(checkfirst=True)
```

this seems to work for me as well (and then the `base = declarative_base()` is not needed)

Reply:
Yes works for me as well. Updated accordingly.